docs/architecture/cloud-native/relational-vs-nosql-data.md

```diff
@@ -68,13 +70,11 @@ Based upon specific data requirements, a cloud-native-based microservice can imp
 
 |  Consider a NoSQL datastore when: | Consider a relational database when: |
 | :-------- | :-------- |
-| You have high volume workloads that require large scale | Your workload volume is consistent and requires medium to large scale |
-| Your workloads don't require ACID guarantees |  ACID guarantees are required |
-| Your data is dynamic and frequently changes | Your data is predictable and highly structured |
-| Data can be expressed without relationships | Data is best expressed relationally |  
-| You need fast writes and write safety isn't critical | Write safety is a requirement |  
-| Data retrieval is simple and tends to be flat | You work with complex queries and reports|
-| Your data requires a wide geographic distribution | Your users are more centralized |
+| You have high volume workloads that require predictable latency at large scale (e.g. latency measured in milliseconds while performing millions of transactions per second) | Your workload volume generally fits within thousands of transactions per second |
```

Comment:
Please add line two back in: The ACID guarantees. Let's add change with the ACID guarantees line added back.

Reply:
ACID is not a differentiator between NoSQL vs RDBMS per above. I'm guessing this was a miss in which the original author hasn't had experience with modern NoSQL databases created in the last 10 years.

Reply:
See the reader's point of view and agree on the change. But, again, disappointed by the sarcastic, demeaning tone.

Reply:
I stand corrected. Totally misunderstood the context. @aliuy explained the concept of gen1 and gen2 NoSQL datastores. Excited that @aliuy is bringing timely information and a great deal of technical and architectural depth to the book. Kudos to @aliuy. Stayed tuned for some great updates.